recipes-nemomobile/sensorfw/sensorfw-hybris-hal-plugins_git.bb

```diff
@@ -0,0 +1,18 @@
+require sensorfw.inc
+
+EXTRA_QMAKEVARS_PRE = "CONFIG+=autohybris"
```

Comment:
In that case we should *always* want hybris so maybe we should use CONFIG+=hybris instead ? Iirc this would avoid dynamically checking if hybris is enabled which would silently avoid compiling the hybris plugin if hybris is not detected.

Reply:
Good idea, but won't work for now since the `hybris` target fails to compile currently.
Is it ok for you if I fix this in a next PR (https://github.com/AsteroidOS/meta-asteroid/pull/98 but using the current work as the base)?

Reply:
Now integrated in this PR.